src/modules/cpu/kernel/glitch.hpp

```diff
@@ -0,0 +1,1177 @@
+                rpp_simd_load(rpp_load48_u8pln3_to_f32pln3_avx, (Rpp8u *)srcPtrTempR, (Rpp8u *)srcPtrTempG, (Rpp8u *)srcPtrTempB, p);
+                rpp_simd_store(rpp_store48_f32pln3_to_u8pkd3_avx, (Rpp8u *)dstPtrTemp, p);
+            }
+            else if( typeid(T) == typeid(Rpp8s)){
+                __m256 p[6];
+                rpp_simd_load(rpp_load48_i8pln3_to_f32pln3_avx, (Rpp8s *)srcPtrTempR, (Rpp8s *)srcPtrTempG, (Rpp8s *)srcPtrTempB, p);
+                rpp_simd_store(rpp_store48_f32pln3_to_i8pkd3_avx, (Rpp8s *)dstPtrTemp, p);
+            }
+            srcPtrTempR += 16;
+            srcPtrTempG += 16;
+            srcPtrTempB += 16;
+            dstPtrTemp += 48;
+        }
+        srcPtrRowR += srcDescPtr->strides.hStride;
+        srcPtrRowG += srcDescPtr->strides.hStride;
+        srcPtrRowB += srcDescPtr->strides.hStride;
+        dstPtrRow += dstDescPtr->strides.hStride;
+    }
+}
+
+template <typename T> void convert_Pkd3_Pln3( T *srcPtrChannel, T *dstPtrChannel, RpptDescPtr srcDescPtr, RpptDescPtr dstDescPtr, int width)
+{
+    T *srcPtrRow, *dstPtrRowR, *dstPtrRowG, *dstPtrRowB;
+    srcPtrRow = srcPtrChannel;
+    dstPtrRowR = dstPtrChannel;
+    dstPtrRowG = dstPtrRowR + dstDescPtr->strides.cStride;
+    dstPtrRowB = dstPtrRowG + dstDescPtr->strides.cStride;
+
+    for(int i = 0; i < srcDescPtr->h; i++)
+    {
+        T *srcPtrTemp, *dstPtrTempR, *dstPtrTempG, *dstPtrTempB;
+        srcPtrTemp = srcPtrRow;
+        dstPtrTempR = dstPtrRowR;
+        dstPtrTempG = dstPtrRowG;
+        dstPtrTempB = dstPtrRowB;
+
+        int vectorLoopCount = 0;
+        for (; vectorLoopCount < width; vectorLoopCount += 48)
+        {
+            if(typeid(T) == typeid(Rpp8u))
+            {
+                __m256 p[6];
+                rpp_simd_load(rpp_load48_u8pkd3_to_f32pln3_avx, (Rpp8u *)srcPtrTemp, p);
+                rpp_simd_store(rpp_store48_f32pln3_to_u8pln3_avx, (Rpp8u *)dstPtrTempR, (Rpp8u *)dstPtrTempG, (Rpp8u *)dstPtrTempB, p);
+            }
+            else if(typeid(T) == typeid(Rpp8s)){
+                __m256 p[6];
+                rpp_simd_load(rpp_load48_i8pkd3_to_f32pln3_avx, (Rpp8s *)srcPtrTemp, p);
+                rpp_simd_store(rpp_store48_f32pln3_to_i8pln3_avx, (Rpp8s *)dstPtrTempR, (Rpp8s *)dstPtrTempG, (Rpp8s *)dstPtrTempB, p);
+            }
+            srcPtrTemp += 48;
+            dstPtrTempR += 16;
+            dstPtrTempG += 16;
+            dstPtrTempB += 16;
+        }
+        srcPtrRow += srcDescPtr->strides.hStride;
+        dstPtrRowR += dstDescPtr->strides.hStride;
+        dstPtrRowG += dstDescPtr->strides.hStride;
+        dstPtrRowB += dstDescPtr->strides.hStride;
+    }
+}
+
+RppStatus glitch_u8_u8_host_tensor(Rpp8u *srcPtr,
+                                   RpptDescPtr srcDescPtr,
+                                   Rpp8u *dstPtr,
+                                   RpptDescPtr dstDescPtr,
+                                   Rpp32u *batch_x_offset_r,
```

Comment:
Don't use batch_ for the variables, you can have it as xOffsetR, yOffsetR applicable all the offset variables in the functions definitions in this file